nix/pkgs/rusty_jwt_tools_ffi/crate2nix-sources

```diff
@@ -0,0 +1 @@
+/nix/store/ygjyja4x8isvc0ja306kvfwz3ikmdqk8-crate2nix-sources
```

Comment:
What does this do? I'm a bit suspicious every time I see a store path somewhere referenced directly...

Reply:
We don't want crate2nix-sources checked into the repository, this is still a draft due to the upstream crate2nix issue linked above.

If you follow the readme instructions, you should end up with the same store path the symlink here is referring to, and you can at least build locally.

Once the issue is resolved, we'll probably add these to .gitignore.

Reply:
Alright. Will you keep this in mind or should we track this somewhere? (The open work)

Reply:
Already tracked in WPB-5294.

Reply:
perfect, I think I'm fine with this then.